mlrun/runtimes/databricks/databricks_wrapper.py

```diff
@@ -27,26 +27,26 @@
 
 def run_mlrun_databricks_job(
     context,
-    mlrun_internal_code,
-    mlrun_internal_token_key="DATABRICKS_TOKEN",
-    mlrun_internal_timeout_minutes=20,
-    mlrun_internal_number_of_workers=1,
+    task_parameters: dict,
     **kwargs,
 ):
+    spark_app_code = task_parameters["spark_app_code"]
+    token_key = task_parameters.get("token_key", "DATABRICKS_TOKEN")
```

Comment:
```suggestion
    token_key = task_parameters.get("token_key", "REPLACE_ME_DATABRICKS_TOKEN")
```
kind of a token?

Reply:
That's the default name of the secret. It's an access token. This PR doesn't change it, only the way that the parameters are passed.